drivers/philips-jointspace/device.js

```diff
@@ -119,6 +124,48 @@ class PhilipsTV extends Homey.Device {
             // Just return true, it takes way too long to wait for a response
             return true;
         });
+
+        this.registerCapabilityListener('volume_up', value => {
+            this.sendKey('VolumeUp');
+            return Promise.resolve();
```

Comment:
Wouldn't you want to return the promise returned by `this.sendKey` instead of always resolving? 

```suggestion
            return this.sendKey('VolumeUp');
```

Or does always resolving prevent some inintended behavior that I'm not seeing here?

Reply:
It's possibel... honestly, I'm not that skilled with node / js promises, so this was a solution I took from other similar app.